app/src/main/res/layout/fragment_map.xml

```diff
@@ -48,7 +81,7 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         android:text="@string/gps_accuracy_double_colon"
-        android:textColor="@color/red"
+        android:textColor="@color/toolbar"
```

Comment:
The toolbar color should only be used for the toolbar. A new color can be used instead, if needed.

Reply:
My bad, must have creeped when I was mucking with the design tool.